app/helpers/posts_helper.rb

```diff
@@ -8,4 +8,8 @@ def authorized_upvote(upvote)
 
     'login'
   end
+
+  def image_included(img)
```

Comment:
method name needs to be relevant to what happens inside the method. In this method you are checking the image format to be png or jpg right? but the name means something else